src/main/java/net/digimonworld/decodetools/res/payload/xtvo/XTVOVertex.java

```diff
@@ -61,6 +61,54 @@ public ByteBuffer write() {
         return buff;
     }
     
+    public Float getX() {

```

Comment:
```suggestion
    public float getX() {

```

return primitive types, also applies to all the other cases